doc/modules/model_evaluation.rst

```diff
@@ -11,13 +11,16 @@ predictions:
 
 * **Estimator score method**: Estimators have a ``score`` method providing a
   default evaluation criterion for the problem they are designed to solve.
-  This is not discussed on this page, but in each estimator's documentation.
+  Most commonly this is mean :ref:`accuracy <accuracy_score>` for classifiers and the
```

Comment:
Yeah good pick up, I have also noticed that we use "mean accuracy" and "accuracy" somewhat inconsistently in our docs.

My initial thought was that we use "mean", for multi-label cases, but I realise that is not the case.

From our docs 'accuracy' is "either the fraction (default) or the count (normalize=False) of correct predictions." so I think by definition its 'over all samples in the test set'. (and if we are returning count, there is no averaging here at all).

I would vote for removing 'mean' where we are referring to the accuracy score.

Reply:
Oh, I just understood better what "mean accuracy" refers to: when we use re-sampling like in a cross-validation and pass 'accuracy_score' as an argument into the `scoring` param, we return the mean over all the accuracy scores archived over all the `k` folds instead of returning all the accuracy scores separately.

It is not what I have thought in the beginning.